web-common/src/features/dashboards/time-controls/SelectorButton.svelte

```diff
@@ -9,9 +9,9 @@
 <button
   {disabled}
   class:bg-gray-200={active}
-  class="px-3 py-2 rounded grid gap-x-2 {active
+  class="px-3 py-2 rounded grid gap-x-2 {!disabled
     ? 'hover:bg-gray-200'
-    : ''} {active ? 'hover:dark:bg-gray-600' : ''} items-center"
+    : ''} {!disabled ? 'hover:dark:bg-gray-600' : ''} items-center"
```

Comment:
Can we combine the two ternary operators? `{!disabled ? 'hover:bg-gray-200 hover:dark:bg-gray-600' : ''}`